src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,35 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  static checkAlive() {
+    this.alive = this.alive.filter(animal => animal.health > 0);
+  }
+
+  health = 100;
+
+  constructor(name) {
+    this.name = name;
+    Animal.alive.push(this);
+  }
```

Comment:
It is better to move health initialisation here if you have constructor.
```suggestion
  constructor(name) {
    this.name = name;
    this.health = 100;
    Animal.alive.push(this);
  }
```